<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="ukk" stdset="0">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="gB_ListaWynikow">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>241</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>LISTA WYNIKOW</string>
    </property>
    <widget class="QPlainTextEdit" name="pt_ListaWynikow">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>221</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="rB_Czas">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>CZAS</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rB_Kroki">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>KROKI</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gB_Przyciski">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>240</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="bt_ZapiszGre">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ZAPISZ GRE DO PLIKU</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_LiczbaWierszy">
     <property name="geometry">
      <rect>
       <x>148</x>
       <y>33</y>
       <width>71</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_EkranStartowy">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>EKRAN STARTOWY</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_WybranyGracz">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>10</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>LICZBA WIERSZY</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_LiczbaWierszy">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>40</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>LICZBA WIERSZY</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_UlozAutomatycznie">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ULOZ AUTOMATYCZNIE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_NowaUkladanka">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>NOWA GRA</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_WczytajGre">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>WCZYTAJ GRE Z PLIKU</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gB_Info">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>200</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="lb_LiczbaKrokow">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>43</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>LICZBA WIERSZY</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CZAS:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_Czas">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>LICZBA WIERSZY</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>43</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>KROKI:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1066</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <signal>signal1()</signal>
  <slot>slot1()</slot>
  <slot>slot2()</slot>
 </slots>
</ui>
